src/lib/transaction_snark/transaction_snark.ml

```diff
@@ -2613,6 +2619,7 @@ module Base = struct
                     in
                     Amount.Signed.create_var ~magnitude ~sgn:Sgn.Checked.neg
                   in
+                  new_account_fee := account_creation_fee ;
```

Comment:
reviewer: please confirm this is the proper locus to determine the account creation fee